cms/templates/home_page.html

```diff
@@ -10,16 +11,29 @@
 {% endblock %}
 
 {% block content %}
-    <div class="hero-container" style="background-image: url('{% wagtail_img_src page.hero%}')">
-      <div class="container hero-content">
-        <div class="hero-title">
-          {% if page.hero_title %}
-            <h1>{{ page.hero_title }}</h1>
-          {% endif %}
-          {% if page.hero_subtitle %}
-            <h2>{{ page.hero_subtitle }}</h2>
-          {% endif %}
-        </div>
-      </div>
-    </div>
-{% endblock %}
\ No newline at end of file
+<div class="course-section">
+  <div class="container">
+    {% if product_cards_section_title %}
+      <h1>{{ product_cards_section_title }}</h1>
+    {% endif %}
+    <ul class="course-listing">
+      {% for card in product_cards %}
+      <li>
+        <a href="#">
+          <img src="{% wagtail_img_src card.feature_image %}" alt="Featured Product Image">
+          <div class="course-info">
+            <h2 class="course-card-title">{{ card.title }}</h2>
+            <div class="course-card-description">
+              {{ card.description |richtext }}
+            </div>  
+            <span class="date">
+              Start date: {{ card.product.first_unexpired_run.start_date |date:"F j, Y" }}
+            </span>
+          </div>
+        </a>
+      </li>
+      {% endfor %}
+    </ul>
```

Comment:
Corresponding change to the suggestion I made for the model `get_context` method above. A couple extra changes I made:
- I changed the `<a>` to `<div class="course-info">` because this element isn't linking to anything, so it shouldn't be a link. You can update the css selector accordingly
- I removed the "start date" section if there is no start date to show, which is something we should _always_ assume is the desired behavior

```suggestion
    <ul class="course-listing">
      {% for product in products %}
      <li>
        <div class="course-card">
          <img src="{% wagtail_img_src product.feature_image %}" alt="Featured course image">
          <div class="course-info">
            <h2 class="course-card-title">{{ product.title }}</h2>
            <div class="course-card-description">
              {{ product.description |richtext }}
            </div>
            {% if product.start_date %}
              <span class="date">
                Start date: {{ product.start_date | date:"F j, Y" }}
              </span>
            {% endif %}
          </div>
        </div>
      </li>
      {% endfor %}
    </ul>
```

Reply:
If it is not the link then How the user be able to get the product detail page ?

Reply:
@asadiqbal08 Sorry, I got this confused with the faculty section which will not have links. We have the product detail page now, so you should link to it from these cards. I think you should be able to use Wagtail's `pageurl` for this (https://docs.wagtail.io/en/stable/topics/writing_templates.html#pageurl)